### PR TITLE
Add spinner to new steps

### PR DESCRIPTION
**Pull request by kateinkim**
Addresses https://github.com/infinitered/ignite/issues/1619

Adds loading spinner to ignite new command steps (with emojis).

Demo @ 60x speed:

https://user-images.githubusercontent.com/53795920/115897204-688ada00-a419-11eb-8605-8b8327ff3a79.mov

### CODE REVIEW

File: src/tools/pretty.ts [hunk 1]
```diff
@@ -1,12 +1,36 @@
-import { print } from "gluegun/print"
+import { print } from "gluegun"
```
Comment (jamonholmgren) [inline src/tools/pretty.ts:1]: Was this not working with `gluegun/print`? If so, that might be a bug with Gluegun.

File: src/tools/pretty.ts [hunk 2]
```diff
@@ -1,12 +1,36 @@
...
 
 const { cyan, gray, white, bold, red, yellow } = print.colors
 const { underline } = print.colors
 
+type Spinner = ReturnType<typeof print.spin>
+const spinners: { [key: string]: Spinner } = {}
```
Comment (jamonholmgren) [inline src/tools/pretty.ts:7]: I like the idea of having multiple spinners at once and tracking them.

**Review by jamonholmgren — APPROVED** [review]
This looks great to me. Nice job Kate! Excited to finally have nice spinners in the new version of Ignite!

File: src/tools/pretty.ts (hunk 1 above)
Comment (kateinkim) [reply src/tools/pretty.ts:1]: It works but is untyped. I can switch it back but I thought it was nice to have the type safety.

Comment (jamonholmgren) [reply src/tools/pretty.ts:1]: Ooh, didn't realize it was untyped!! That's something we'll want to fix eventually on Gluegun. Thanks for pointing it out!

**Pull request merged**